gatsby-node.js

```diff
@@ -317,4 +346,8 @@ function warning(message) {
 	Log.warning(message);
 }
 
+function sleep(ms) {
+	return new Promise((res) => setTimeout(res, ms));
+}
```

Comment:
This is the right approach for this PR, but if you're targeting newer versions of Node (18+ only), you can actually do:

```js
import { setTimeout as sleep } from 'node:timers/promises';

await sleep(1000);
```

😄